src/urbit/sur/chess.hoon

```diff
@@ -13,34 +17,56 @@
       %queen
       %king
   ==
+::
+::  a pawn can be promoted to...
 +$  chess-promotion
   $~  %queen
   $?  %knight
       %bishop
       %rook
       %queen
   ==
+::
+::  a chess piece is a cell of its side and type
 +$  chess-piece
   $~  [%white %pawn]
   $:  chess-side
       chess-piece-type
   ==
+::
+::  a chessboard has eight rows called ranks, numbered %1 to %8
 +$  chess-rank
   $~  %1
   ?(%1 %2 %3 %4 %5 %6 %7 %8)
+::
+::  a chessboard has eight columns called files, labelled %a to %h
 +$  chess-file
   $~  %a
   ?(%a %b %c %d %e %f %g %h)
+::
+::  a chess square is a cell of file and rank
 +$  chess-square
   $~  [%a %1]
   [chess-file chess-rank]
-+$  chess-piece-on-square
-  [square=chess-square piece=chess-piece]
```

Comment:
Did you ask to move this below `chess-board`? I might have moved it because it's more intuitive for the reader to refer to "key-value pairs from chess-board" after `chess-board`.

Reply:
I think I did. Regardless of who thought of it, it's the correct thing to do, for the reason that you list.